.travis.yml

```diff
@@ -21,53 +23,108 @@ addons:
 
 env:
     global:
+
         # The following versions are the 'default' for tests, unless
-        # overidden underneath. They are defined here in order to save having
+        # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
+        - PYTHON_VERSION=3.6
+        - NUMPY_VERSION=stable
```

Comment:
Yes, the LTS build (astropy 1.0) doesn't exist with Numpy 1.13. Maybe you can set Numpy version to 1.12 just for the LTS builds. I'll look at the dev failures.